cid/cli.py

```diff
@@ -135,13 +137,17 @@ def export(ctx, **kwargs):
     
     \b
     Command options:
-        --analysis-name       Analysis you want to share (not needed if analysis-id is provided).
-        --analysis-id         ID of analysis you want to share (open analysis in browser and copy id from url)
-        --template-id         Template Id
-        --dashboard-id        Target Dashboard Id
-        --template-version    Version description vX.Y.Z
-        --reader-account      Account id with howm you want to share or *
-        --output              A filename (.yaml)
+        --analysis-name              Analysis you want to share (not needed if analysis-id is provided).
+        --analysis-id                ID of analysis you want to share (open analysis in browser and copy id from url)
+        --template-id                Template Id
+        --dashboard-id               Target Dashboard Id
+        --template-version           Version description vX.Y.Z
+        --reader-account             Account id with howm you want to share or *
+        --dashboard-export-method
+               (definition|template) A method (definition=pull json definition of Analysis OR template=create QuickSught Teamplate)
+        --export-known-datasets
```

Comment:
In most cases it will be dashboards created on top of existing. So we do not need to override default behaviour. For example most of datasets are dependent on account_map. We do not want to override account map with a concrete implementation in source account.